src/main/scala/org/broadinstitute/hail/stats/package.scala

```diff
@@ -2,9 +2,43 @@ package org.broadinstitute.hail
 
 import org.apache.commons.math3.distribution.HypergeometricDistribution
 import org.broadinstitute.hail.utils._
+import org.broadinstitute.hail.variant.Genotype
 
 package object stats {
 
+  def imputeInfoScore(gs: Iterable[Genotype]) = {
```

Comment:
I kinda liked the combiner.  I see it having these advantages:

* can convert it to an annotation at the end with a class method
* can put the annotation schema on a companion object
* can have its merge function take Option[Array[Double]], which makes it totally general and somebody else could pick up that piece of code very easily
* supports aggregation rather than just folding, which helps to future-proof in case our representation ever distributes across samples as well, and follows the spark Combiner model

I would've put the combiner/companion object into a new file in the stats package.  That said, if you're happy with this structure, that's fine